components/shared/responsiveimage.tsx

```diff
@@ -13,7 +13,7 @@ export const ResponsiveImage: React.FC<{
   blur?: boolean;
   urlBuilder?: UseNextSanityImageBuilder;
 }> = ({ image, alt, onClick, priority, layout = "fill", urlBuilder, blur = true }) => {
-  const imageProps = useNextSanityImage({ clientConfig: config }, image, {
+  const imageProps = useNextSanityImage({ clientConfig: projectConfig }, image, {
```

Comment:
Are we certain this is safe? For example, in production, the client configuration `useCdn` option is `true`, but undefined in `projectConfig` – could this affect `useNextSanityImage` behaviour?

Reply:
No, I've switched it out with the sanity client object now, so it should be okay 👍 This is the suggested solution in the documentation.